dotnet/samples/Concepts/PromptTemplates/LiquidPrompts.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
nit: We should add link to this class in README file here:
https://github.com/microsoft/semantic-kernel/blob/9e70e7172b073851d8843bb37ffff766a84ec0e8/dotnet/samples/Concepts/README.md?plain=1#L116-L122

Can be added later.

Reply:
Follow up task https://github.com/microsoft/semantic-kernel/issues/6166